src/OpenTelemetry.Exporter.InMemory/InMemoryExporterOptions.cs

```diff
@@ -14,12 +14,9 @@
 // limitations under the License.
 // </copyright>
 
-using System.Collections.Generic;
-
 namespace OpenTelemetry.Exporter
 {
     public class InMemoryExporterOptions
     {
-        public ICollection<object> ExportedItems { get; set; }
```

Comment:
Can this whole class go away now?

Reply:
Yeah we could get rid of this class. I kept it just so if we add options for InMemoryExporter later on we could use this. We could always add the class later on as well.  Should I just delete it?

Reply:
yea deletre it. If we want to add any options, then add it then.

Reply:
I have deleted this file.